source/compatibility.tex

```diff
@@ -1969,7 +1969,7 @@
 }
 \end{codeblock}
 
-ISO C will accept this usage of pointer to void being assigned
+ISO C will accept this usage of pointer to \tcode{void} being assigned
 to a pointer to object type.
 \Cpp{} will not.
```

Comment:
"will accept" -> "accepts", "will not" -> "does not", while we're fixing things here.